exercises/en/test_06_21.py

```diff
@@ -0,0 +1,17 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    import pandas as pd
+    # If an assertion fails, the message will be displayed
+    assert sorted(numeric_features) ==  ['age', 'sitting_hrs'], "The numeric features are incorrect. Please try again."
+    assert sorted(binary_features) == ['accident_trauma', 'childish_diseases', 'surgical_intervention'], "The binary features are incorrect. Please try again."
+    assert sorted(ordinal_features) == ['freq_alcohol_con', 'high_fevers_last_year', 'smoking_habit'], "The ordinal features are incorrect. Please try again."
+    assert sorted(categorical_features)  == ['season'], "The categorical features are incorrect. Please try again."
+    assert fever_order == ['no', 'more than 3 months ago', 'less than 3 months ago'], "The order for fever is incorrect. Please try again."
+    assert smoking_order == [ 'never', 'occasional', 'daily'], "The order for smoking is incorrect. Please try again."
+    assert alcohol_order[0:3] == ['hardly ever or never', 'once a week', 'several times a week'], "The order for alcohol is incorrect. Please try again."
+    assert min(pd.DataFrame(scores)['test_score']) > 0.80 and max(pd.DataFrame(scores)['test_score']) < 0.90, "The range of your test scores is incorrect. Are you calling the cross_validate function?"
```

Comment:
Can you check that each of the pipelines are correctly assigned? I am fairly confident you can check the attributes.